src/angulartics-ga.js

```diff
@@ -92,5 +92,21 @@ angular.module('angulartics.google.analytics', ['angulartics'])
 
   });
 
+  $analyticsProvider.registerExceptionTrack(function (properties) {
+    if (!properties || !properties.appId || !properties.appName || !properties.appVersion) {
+        console.error('Must set appId, appName and appVersion.');
+        return;
+    }
+
+    if(!properties.fatal) {
```

Comment:
This way I will never be able to send `fatal` as `false`, which I don't think is what we should to here. Maybe change to
`if (properties.fatal == undefined) {`?